recipes/cppbenchmark/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+patches:
+  "8605a8e886647e964180cb7a622424404a1da01f":
+    - patch_file: "patches/00001-update-cmakelists.patch"
+      base_path: "source_subfolder"
+sources:
+  "8605a8e886647e964180cb7a622424404a1da01f":
```

Comment:
Use the official release 1.0.0.0: https://github.com/chronoxor/CppBenchmark/releases/tag/1.0.0.0

Reply:
Version 1.0.0.0 added in 7d13162f